backends/ebpf/runtime/ebpf_kernel.h

```diff
@@ -23,30 +23,48 @@ limitations under the License.
 #ifndef BACKENDS_EBPF_RUNTIME_EBPF_KERNEL_H_
 #define BACKENDS_EBPF_RUNTIME_EBPF_KERNEL_H_
 
-#include <linux/pkt_cls.h>  // TC_ACT_OK, TC_ACT_SHOT
-#include <string.h>         // memset()
-#include <unistd.h>         // syscall()
-#include <sys/syscall.h>    // __NR_bpf
 #include "ebpf_common.h"
 
 
+/* If we operate in user space we only need to include bpf.h and
```

Comment:
Will this compile with any kernel version?

Reply:
Well this is what libbpf says: "No ties to any specific kernel, transparent handling of older kernels. Libbpf is designed to be kernel-agnostic and work across multitude of kernel versions. It has built-in mechanisms to gracefully handle older kernels, that are missing some of the features, by working around or gracefully degrading functionality. Thus libbpf is not tied to a specific kernel version and can/should be packaged and versioned independently." 

I try to use their custom headers everywhere I can.